mesheryctl/internal/cli/root/system/error.go

```diff
@@ -25,23 +25,23 @@ const (
 )
 
 func ErrHealthCheckFailed(err error) error {
-	return errors.New(ErrHealthCheckFailedCode, errors.Alert, []string{"Health checks failed"}, []string{err.Error()}, []string{}, []string{})
+	return errors.New(ErrHealthCheckFailedCode, errors.Alert, []string{"Health checks failed"}, []string{err.Error()}, []string{"health checks execution failed"}, []string{"health checks execution should passed to start Meshery server successfully"})
 }
 
 func ErrInvalidAdapter(err error, obj string) error {
 	return errors.New(ErrInvalidAdapterCode, errors.Alert, []string{"Invalid adapter ", obj, " specified"}, []string{err.Error()}, []string{}, []string{})
 }
 
 func ErrDownloadFile(err error, obj string) error {
-	return errors.New(ErrDownloadFileCode, errors.Alert, []string{"Error downloading file ", obj}, []string{err.Error()}, []string{}, []string{})
+	return errors.New(ErrDownloadFileCode, errors.Alert, []string{"Error downloading file ", obj}, []string{err.Error()}, []string{"failed downloading current docker-compose file"}, []string{"Make sure docker-compose file is downloaded successfully"})
```

Comment:
@nehaLohia27 I thought `ErrDownloadFile` was a generic error for all types of files? Is it supposed to be specifically intended for docker compose files?

Reply:
@metonymic-smokey ErrDownloadFile is only for docker compose or operator manifest file in the code. i have made changes according to this. can you please check once

Reply:
yes this looks better @nehaLohia27 !

Reply:
okay @metonymic-smokey . mesheryctl build and release is failing. can u check once ?  i m not sure why the build failed